<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>surface</class>
 <widget class="QMainWindow" name="surface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>282</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>280</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Тема</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Градиенты</string>
     </property>
     <widget class="QMenu" name="menu_3">
      <property name="title">
       <string>Выбрать градиент</string>
      </property>
      <addaction name="chkdCold"/>
      <addaction name="chkdWarm"/>
      <addaction name="chkdNeon"/>
     </widget>
     <addaction name="useGrad"/>
     <addaction name="menu_3"/>
    </widget>
    <addaction name="menu_2"/>
    <addaction name="pckMClr"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="useGrad">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Использовать градиент?</string>
   </property>
  </action>
  <action name="pckMClr">
   <property name="text">
    <string>Выбор основного цвета</string>
   </property>
   <property name="toolTip">
    <string>Выбор цвета сплошного отображения</string>
   </property>
  </action>
  <action name="chkdCold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Холодный</string>
   </property>
  </action>
  <action name="chkdWarm">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Теплый</string>
   </property>
  </action>
  <action name="chkdNeon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Неоновый</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
